src-java/kilda-pce/src/test/java/org/openkilda/pce/impl/AvailableNetworkTest.java

```diff
@@ -222,7 +211,7 @@ public void shouldSetEqualCostForPairedLinks() {
     }
 
     @Test
-    public void shouldCreateSymmetricOutgoingAndIncomming() {
+    public void createSymmetricOutgoingAndIncomingTest() {
```

Comment:
Could you please add a word link to the method's name? It's outgoing and incoming links

Reply:
Sure, will do. Thanks!